src/main/java/pl/miloszkajetan/serwisAukcyjny/user/UserRoleRepository.java

```diff
@@ -0,0 +1,14 @@
+package pl.miloszkajetan.serwisAukcyjny.user;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRoleRepository extends JpaRepository<UserRole, Long> {
+
+    @Query(" SELECT ur FROM UserRole ur WHERE ur.userRoleName = ?1 ") //co oznacza ? przy tej 1?
```

Comment:
pierwszy parametr, to tylko format zapisu który sobie ktoś z projektantów Spinga wymyślił.